themes/inbeat/assets/scss/pages/_home.scss

```diff
@@ -131,15 +131,10 @@
 }
 
 #hero {
-  padding-top: 8.5rem;
-  padding-bottom: 6rem;
+  padding-top: 10rem;
+  padding-bottom: 5rem;
```

Comment:
I think the goal with previous CSS was to achieve kind of full screen. You decide what you prefer, up to you guys!

Reply:
Yeah, it was buggy - For now I'll just set a fix value and adjust later